documentation/asciidoc/computers/raspberry-pi/boot-eeprom-rpi4.adoc

```diff
@@ -1,6 +1,6 @@
 == Raspberry Pi 4 Boot EEPROM
 
-The Raspberry Pi 4 has an SPI-attached EEPROM (4MBits/512KB), which contains code to boot up the system and replaces `bootcode.bin` previously found in the boot partition of the SD card. Note that if a `bootcode.bin` is present in the boot partition of the SD card in a Pi 4, it is ignored.
+Raspberry Pi 4 and 400 computers use an EEPROM to boot the system. All other models of Raspberry Pi computer use the `bootcode.bin` file located in the boot filesystem.
```

Comment:
And perhaps "use an EEPROM" should be "use code stored in an EEPROM" ?

Reply:
> And perhaps "use an EEPROM" should be "use code stored in an EEPROM" ?

More words for essentially the same meaning.